database/seedToPSQL.js

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Again, I would suggest removing the first line and running this file through a linter if you haven't already